internal-dns/types/tests/output/internal-dns-zone.txt

```diff
@@ -72,61 +72,61 @@ builder: "non_trivial"
     {
       "type": "SRV",
       "data": {
-        "port": 127,
         "prio": 0,
-        "target": "001de000-c04e-4000-8000-000000000002.host.control-plane.oxide.internal",
-        "weight": 0
+        "weight": 0,
+        "port": 127,
+        "target": "001de000-c04e-4000-8000-000000000002.host.control-plane.oxide.internal"
```

Comment:
The reordering here is not important, it happened because we're no longer using progenitor-generated types (which I didn't realize got reordered in alphabetical order!)